src/main/java/no/nav/fo/veilarbregistrering/registrering/resources/InternalRegistreringResendingServlet.java

```diff
@@ -16,7 +16,7 @@ public InternalRegistreringResendingServlet(BrukerRegistreringService brukerRegi
         this.brukerRegistreringService = brukerRegistreringService;
     }
 
-    public static final String PATH = "/internal/registrering/status";
+    public static final String PATH = "/internal/status/endre";
```

Comment:
Hadde det funket å bare brukt `/internal/status` her også, men at det PUT som gjør denne ulik den andre som er GET? Eller skaper det krøll ift Servlet-configen?

Reply:
Usikker..

Men det er ingen dum idé å bruke `internal/status` her. For da blir det naturlig å bruke GET i samme servlet for å hente statusinformasjon til én enkel registering. Så kan vi heller endre pathen for den andre servleten til f.eks `/internal/statusoversikt`

Reply:
Usikker..

Men det er ingen dum idé å bruke `internal/status` her. For da blir det naturlig å bruke GET i samme servlet for å hente statusinformasjon til én enkel registering. Så kan vi heller endre pathen for den andre servleten til f.eks `/internal/statusoversikt`